src/components/Calculator.js

```diff
@@ -1,70 +1,69 @@
-import React from 'react';
 import './Calculator.css';
+import { useState } from 'react';
 import calculate from '../logic/calculate.js';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.handleClickEvent = this.handleClickEvent.bind(this);
-  }
-
-  handleClickEvent(event) {
+function Calculator() {
```

Comment:
[OPTIONAL]
- [ ] In the latest version of JS it's encouraged to use the `arrow function` rather than `regular function declaration`, You may consider taking that into account